WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -3,10 +3,18 @@ class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
   def index
+    @everything_good = true
+
     @jobs_that_should_have_run_by_now = Delayed::Job.where(attempts: 0).where('created_at < ?', MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING.minutes.ago)
     @oldest_job_that_should_have_run_by_now = @jobs_that_should_have_run_by_now.order(:created_at).first
+    if @oldest_job_that_should_have_run_by_now
+      @everything_good = false
+    end
 
-    @everything_good = @oldest_job_that_should_have_run_by_now.nil?
+    @regulations_load_error = Regulation.regulations_load_error
+    if @regulations_load_error
+      @everything_good = false
+    end
```

Comment:
`.nil?` is great if it works. I'm just slightly nervous about `Regulation.regulations_load_error` being changed to return an empty array or even maybe returning false now. With test coverage this doesn't matter, but I'm guessing we don't have that here.

That's probably silly of me. Use `.nil?` if you're confident about it.

I much prefer `&&=` to `false if`